checks/docker/update_instruction_alone.rego

```diff
@@ -24,23 +24,24 @@ deny[res] {
 
 	command = concat(" ", run.Value)
 
+	is_package_manager(command)
 	is_valid_update(command)
 	not update_followed_by_install(command)
 
 	msg := "The instruction 'RUN <package-manager> update' should always be followed by '<package-manager> install' in the same RUN statement."
 	res := result.new(msg, run)
 }
 
-is_valid_update(command) {
-	chained_parts := regex.split(`\s*&&\s*`, command)
-
-	array_split := split(chained_parts[_], " ")
+package_manager_regex := `(apk)|(apt-get)|(yum)`
 
-	len = count(array_split)
+is_package_manager(command) {
+	regex.match(package_manager_regex, command)
+}
 
-	update := {"update", "--update"}
+update_regex := `( update)|( check-update)`
 
-	array_split[len - 1] == update[_]
+is_valid_update(command) {
+	regex.match(update_regex, command)
```

Comment:
When using regex, we will not be able to establish that the update command refers to the package manager. Example of a command that causes a false positive:`apt-get build-dep && /bin/sh /scripts/someScript.sh update`. Maybe we should split the command by && and check each part?